devbot/bot.py

```diff
@@ -8,6 +8,7 @@
 import devbot.commands
 from devbot.registry import COMMAND_DICT, safe_call, CommandNotFoundError
 from devbot.tools.logging import get_logger
+from devbot.tools.filewrapper import FileWrapper
```

Comment:
It's usually better to import the whole module, so you can see where the class is coming from in the rest of the code. Maybe shorten it though, since `devbot.tools.filewrapper` is quite a mouthful.